File: repaso.py
# num1 = float(input("Ingrese el primer numero: "))
# num2 = float(input("Ingrese el segundo numero: "))
# operacion = input("Ingrese la operacion (+, -, *, /): ")

# if operacion == "+":
#     resultado = num1+num2
# elif operacion == "-":
#     resultado = num1-num2
# elif operacion == "*":
#     resultado= num1*num2
# elif operacion=="/" and num2 !=0:
#     resultado= num1/num2
# else:
#     resultado = "Operacion no valida"

# print(f"Resultado {resultado}")

# contador = 0

# while True:
#     numero = int(input("Ingrese un numero: "))
#     if numero<0:
#         break
#     contador+=1

# print(f"Cntidad de numero positivos ingresados: {contador}")

# numero= int(input("Ingrese un numero: "))
# factorial=1

# for i in range(1,numero+1):
#     factorial *=i

# print(f"Factorial: {factorial}")

# inicio= int(input("Ingrese el numero inicial: "))
# fin= int(input("Ingrese el numero final: "))

# for i in range(inicio,fin+1):
#     if i % 2 ==0:
#         print(i)

import random

numero_secreto = random.randint(1,50)

while True:
    intento = int(input("Ingrese el numero: "))

    if intento == numero_secreto:
        print("Adivinaste")
        break
    elif intento > numero_secreto:
        print("Muy alto")
    else:
        print("Muy Bajo")

File: repaso.js
const prompt = require('prompt-sync')();

// let num1 = parseFloat(prompt("Ingrese el primer numero: "))
// let num2 = parseFloat(prompt("Ingrese el segundo numero: "))
// let operacion = prompt("Ingrese la operacion (+, -, *, /)")

// let resultado;

// switch(operacion){

//     case "+":
//         resultado = num1+num2
//         break;
//     case "-":
//         resultado = num1-num2
//         break;
//     case "*":
//         resultado=num1*num2
//         break
//     case "/":
//         resultado= num2 !==0 ? num1/num2 : "no se puede dividir por 0"
//         break;
//     default:
//         resultado= "Operacion no valida"
// }

// console.log(`Resultado ${resultado}`)

// let contador = 0

// while(true){
//     let numero = parseInt(prompt("Ingrese un numero: "))
//     if(numero<0) break;
//     contador ++;
// }

// console.log("Cantidad de numeros positivos ingresador: ",contador)

// let numero = parseInt(prompt("Ingrese un numero: "))
// let factorial =1;

// for(let i=1; i<=numero;i++){
//     factorial*=i;
// }

// console.log(factorial)


// let inicio = parseInt(prompt("Ingrese el numero Inicial: "))
// let fin = parseInt(prompt("Ingrese el numero final: "))

// console.log("Numero Pares: ")
// for(let i =inicio; i<=fin; i++){
//     if(i%2 ===0){
//         console.log(i)
//     }
// }

let numeroSecreto = Math.floor(Math.random()*50)+1
let intento;

console.log("Adivina el numero entre 1 y 50")

while(true){
    intento = parseInt(prompt("Ingresa el numero: "))

    if(intento===numeroSecreto){
        console.log("Adivinaste")
        break;
    }else if (intento< numeroSecreto){
        console.log("Muy Bajo")
    }else{
        console.log("Muy alto")
    }
}